.bcr/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform: [ "macos", "ubuntu2004" ]
+
+tasks:
+  verify_targets_workspace:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    build_flags:
+      - "--enable_bzlmod=false"
+    build_targets:
```

Comment:
Worth noting that we don't actually check tests into the release binary of `rules_kotlin` so there isn't much to run on the Bazel central registry presubmit side.